backend/bitboxbase/handlers/handlers.go

```diff
@@ -61,7 +62,7 @@ func (handlers *Handlers) Uninit() {
 
 func (handlers *Handlers) getBlockInfoHandler(_ *http.Request) (interface{}, error) {
 	handlers.log.Debug("Block Info")
-	return handlers.base.BlockInfo(), nil
+	return jsonp.MustMarshal(handlers.base.BlockInfo()), nil
```

Comment:
MustMarshal panics, and panics are reserved for programming errors / impossible situations.

If base.BlockInfo() however validates the input and converts it to an appropriate Go type first (see above comment), then this is okay :)